src/components/warning-component.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+import colors from '@styles/colors';
+
+enum warningType {
+    Thunderstorm,
+    Strong_wind,
+    Snowstorm,
+    Rain,
+    Fog,
+    Little_wind,
+    hail,
+}
+
+export function Warning(War: warningType) {
+    switch (War) {
+        case warningType.Thunderstorm:
+            return 'Возможна гроза';
+            break;
+        case warningType.Strong_wind:
+            return 'Возможно сильный ветер';
+            break;
+        case warningType.Snowstorm:
+            return 'Возможна метель';
+            break;
+        case warningType.Rain:
+            return 'Возможно дождь';
+            break;
+        case warningType.Fog:
+            return 'Возможно туман';
+            break;
+        case warningType.Little_wind:
+            return 'Возможно слабый ветер';
+            break;
+        case warningType.hail:
+            return 'Возможно град';
+            break;
```

Comment:
Unnecessary breaks, we return from function in some case so next cases won't be checked, also IDE gives unreachable code warning because of it.
It is better to create file constants.ts in utils and place all string constants there, for example: 
![image](https://user-images.githubusercontent.com/52620425/223991050-bcacaed8-5263-4dc7-93ce-ecd1162d357d.png)